packages/dashboard-frontend/static/loader.html

```diff
@@ -17,7 +17,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="shortcut icon" href="/dashboard/assets/branding/favicon.ico">
+  <link rel="shortcut icon">
```

Comment:
@tolusha Are we going to remove the default value "/dashboard/assets/branding/favicon.ico" at all?